app/api/intendedGraduations/create/func.js

```diff
@@ -0,0 +1,22 @@
+const app = require('@/app')
+
+module.exports = async function (context) {
+  let { graduationId } = context.params;
+  console.log(graduationId)
+  let intendedGraduation = await app.models.intendedGraduation.findOne({
+    graduation: graduationId,
+    user: context.user._id,
+    active: true
+  })
+  console.log(intendedGraduation)
+  if(intendedGraduation) throw new errors.BadRequest(`Inteded graduation already selected for this user.`)
+
+
+  const newintendedGraduation = await app.models.intendedGraduation.create({
+      graduation: graduationId,
+      user: context.user
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 6.](https://app.codacy.com/gh/ufabc-next/ufabc-next-server/pullRequest?prid=8555380)
```suggestion
    user: context.user
```